agent/php_pdo.h

```diff
@@ -161,4 +161,9 @@ extern nr_status_t nr_php_pdo_parse_data_source(
 extern void nr_php_pdo_free_data_sources(struct pdo_data_src_parser* parsed,
                                          size_t nparams);
 
+/*
+ * Purpose : Frees reference incremented, transaction global zvals
+ *           that must be cleaned up prior to postdeactivate
+ */
+extern void nr_pdo_rshutdown();
```

Comment:
There's a naming convention that symbols that are PHP runtime aware and are defined in the agent have the prefix of `nr_php_` and the functions that are agnostic of PHP runtime and are defined in axiom have the prefix of `nr_`. This convention helps in reasoning about the code when reading it. Please update the name of this function to `nr_php_pdo_rshutdown`.